src/Microsoft.SqlTools.ServiceLayer/SqlProjects/Contracts/Projects/CloseSqlProject.cs

```diff
@@ -0,0 +1,14 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.SqlProjects.Contracts
+{
+    public class CloseSqlProjectRequest
+    {
+        public static readonly RequestType<SqlProjectParams, SqlProjectResult> Type = RequestType<SqlProjectParams, SqlProjectResult>.Create("sqlprojects/close");
```

Comment:
Shouldn't this be "sqlprojects/closeSqlObjectScript" to be consistent with all the other request endpoints like "sqlprojects/excludeSqlObjectScript", "sqlprojects/deleteSqlObjectScript", etc.

Reply:
To make it more clear these apis are for sql projects and not scripts, might be helpful to change it to be `sqlprojects/closeProject`

Reply:
Sure, I can make that change